FirebaseRemoteConfig/Sources/RCNConfigSettings.m

```diff
@@ -354,8 +354,8 @@ - (NSString *)nextRequestWithUserProperties:(NSDictionary *)userProperties {
                                                                 FIRRemoteConfigAppVersion()]];
   ret = [ret stringByAppendingString:[NSString stringWithFormat:@", app_build:'%@'",
                                                                 FIRRemoteConfigAppBuildVersion()]];
-  ret = [ret stringByAppendingString:[NSString stringWithFormat:@", sdk_version:'%d'",
-                                                                FIRRemoteConfigSDKVersion()]];
+  ret = [ret stringByAppendingString:[NSString stringWithFormat:@", sdk_version:'%@'",
+                                                                FIRRemoteConfigPodVersion()]];
```

Comment:
Assuming you've confirmed the backend can handle a version string that includes periods versus the pure integer that was sent before?

Reply:
yep! updated the internal bug with details